packages/meditrak-app/app/navigation/actions.js

```diff
@@ -66,13 +60,7 @@ export const goToCreateAccount = () => dispatch => {
   );
 };
 
-export const openSurvey = () =>
-  NavigationActions.navigate({
-    routeName: SURVEY_SCREEN,
-    params: {
-      transition: 'SurveyTransition',
```

Comment:
No need this transition param anymore. This was used for previous transition animation